planemo_ext/galaxy/tools/linters/inputs.py

```diff
@@ -23,10 +25,40 @@ def lint_inputs(tool_xml, lint_ctx):
             if "format" not in param_attrib:
                 lint_ctx.warn("Param input [%s] with no format specified - 'data' format will be assumed.", param_name)
         # TODO: Validate type, much more...
+
+    conditional_selects = tool_xml.findall("./inputs//conditional")
+    for conditional in conditional_selects:
+        select = conditional.find('./param[@type="select"]')
+        # Should conditionals ever not have a select?
```

Comment:
It can have a boolean. According to https://github.com/galaxy-iuc/standards/blob/master/docs/best_practices/tool_xml.rst#booleans we do not recommend to use booleans for a conditional but it is possible.